staging/src/k8s.io/apiserver/pkg/util/flowcontrol/metrics/metrics.go

```diff
@@ -88,6 +103,42 @@ var (
 		},
 		[]string{priorityLevel, flowSchema},
 	)
+	windowedRequestCountsMin = compbasemetrics.NewGaugeVec(
```

Comment:
what window is this calculated over?

Reply:
`windowWidth` in `staging/src/k8s.io/apiserver/pkg/server/filters/priority-and-fairness.go`